packages/app/src/components/Common/ImageCropModal.tsx

```diff
@@ -49,6 +49,13 @@ const ImageCropModal: FC<Props> = (props: Props) => {
   const { t } = useTranslation();
   const reset = useCallback(() => {
     if (imageRef) {
+      // Some SVG files may not have width and height properties, causing the render size to be 0x0
+      // Force imageRef to have width and height by create temporary image element then set the imageRef width with tempImage width
+      const tempImage = new Image();
+      tempImage.src = imageRef.src;
+      imageRef.width = tempImage.width;
```

Comment:
Does this have to be always executed even if an imageRef initially have width and height?
Also, what is the meaning for the code below which not seems to be used anywhere after assigning the `src` value
`tempImage.src = imageRef.src;`